adhocracy-plus/assets/scss/_variables.scss

```diff
@@ -138,3 +138,9 @@ $github: #333;
 $utilities-text-colors: #333; // <-- BS inport fix
 $utilities-colors: (); // <-- BS inport fix
 $utilities-text-colors: (); // <-- BS inport fix
+
+// classifications
+$offensive: #FCE5E5 !default;
+$engaging: #EBE8FF !default;
+$factclaiming: #FCE5EC !default;
+$ai: #E7E7F1 !default;
```

Comment:
hexs should all be lowercase not sure why the linter didn't catch that? also in this case I would use colour names as we may want them elsewhere and the grey we can update an existing variable, same with the purple as we already have a `$purple-light` we don't want loads of different colours if we can possible help it!

Reply:
out of curiosity, i used `!default` but actually without knowing what it does. Do you have an explanation what it does. couldn't find any useful information about it @phillimorland ?

Reply:
it just sets a default for that variable name, we load our `variables` file before we load the bootstrap one in style.scss so the defaults we want are set and the bootstrap ones get ignored because there is already one set.